src/Microsoft.DotNet.Build.Tasks/Targets/PerfTesting.targets

```diff
@@ -52,6 +52,12 @@
   <PropertyGroup Condition="'$(RunPerfTestsForProject)' == 'true'">
     <TestCommandLine>$(PerfTestCommandLine)</TestCommandLine>
   </PropertyGroup>
+  
+  <!-- Perf tests require TargetFrameworkVersion=v5.0. If it isn't installed, throw a helpful warning and run unit tests instead -->
+  <Target Name="ValidateFrameworkVersion" BeforeTargets="RestorePackages">
+    <Error Text="To run performance tests, .NET Portable v5.0 must be installed with VS 2015. Installation instructions available at: https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/performance-tests.md"
```

Comment:
We should try and figure out how to add links to things that aren't in the corefx repo for build tools. Eventually I would prefer these type of documents to live in the core/home repo or perhaps in the build tools repo itself.